### PR TITLE
LPS-176655 Asset Publisher staging message is not styled properly

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
[Link to ticket](https://issues.liferay.com/browse/LPS-176655)

## Test Scenario 

* Enable Staging
* Create a Widget Page
* Add an Asset Publisher
* Open the options dropdown in the topper and select "Staging"

**Expected:** The message that appears is well styled. Maybe it should be an alert or something like that

image

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/publish_portlet.jsp [hunk 1]
```diff
@@ -91,7 +91,10 @@ if (!GroupPermissionUtil.contains(permissionChecker, themeDisplay.getScopeGroup(
 
 	<c:choose>
 		<c:when test="<%= Validator.isNotNull(errorMessageKey) %>">
-			<liferay-ui:message key="<%= errorMessageKey %>" />
+			<clay:stripe
```
Comment (ealonso) [inline modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/publish_portlet.jsp:94]: @BarbaraCabrera I think we can add some container-fluid class, to add some padding/margin on both sides, What do you think?

Comment (BarbaraCabrera) [reply modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/publish_portlet.jsp:94]: Hi @ealonso! I have reviewed the design with the designers and as it is a warning next to the top they have chosen a stripe variation. So I think we would not have to add margins/padding to this one 😄

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
ci:forward:force

**Pull request closed without merging**